cfme/tests/openstack/infrastructure/test_relationships.py

```diff
@@ -4,52 +4,50 @@
 results of the relationships
 """
 import pytest
-from utils import testgen
+
 from cfme.infrastructure.provider.openstack_infra import OpenstackInfraProvider
+from utils import testgen, version
+from utils.appliance.implementations.ui import navigate_to
 
 
-pytestmark = [pytest.mark.meta(server_roles='+smartproxy +smartstate')]
+pytestmark = [pytest.mark.meta(server_roles='+smartproxy +smartstate'),
+              pytest.mark.usefixtures("setup_provider_modscope")]
 
 
 pytest_generate_tests = testgen.generate([OpenstackInfraProvider], scope='module')
 
 
-@pytest.mark.usefixtures("setup_provider_modscope")
-def test_assinged_roles(provider):
-    provider.load_details()
-    result = provider.summary.relationships.deployment_roles.value
-
-    assert result > 0
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.7')
+def test_assigned_roles(provider):
+    navigate_to(provider, 'Details')
+    assert int(provider.get_detail('Relationships', 'Deployment Roles')) > 0
 
 
 def test_nodes(provider):
-    provider.load_details()
-    result = provider.summary.relationships.nodes.value
+    navigate_to(provider, 'Details')
     """
     todo get the list of VM's from external resource and compare
     it with result - currently not 0
     """
 
-    assert result > 0
+    assert int(provider.get_detail('Relationships', 'Nodes')) > 0
 
 
 def test_templates(provider):
-    provider.load_details()
-    result = provider.summary.relationships.templates.value
+    navigate_to(provider, 'Details')
     """
     todo get the list of images/templates from external resource and compare
     it with result - currently  bigger than 0
     """
 
-    assert result > 0
+    assert int(provider.get_detail('Relationships', 'Templates')) > 0
 
 
 def test_stacks(provider):
-    provider.load_details()
-    result = provider.summary.relationships.orchestration_stacks.value
+    navigate_to(provider, 'Details')
     """
     todo get the list of tenants from external resource and compare
     it with result - currently not 0
     """
 
-    assert result > 0
+    assert int(provider.get_detail('Relationships', 'Orchestration stacks')) > 0
```

Comment:
**OPTIONAL**
Consider parametrizing on ['Deployment Roles', 'Nodes', 'Templates', 'Orchestration stacks'] since all of these tests are the same and only the 2nd arg to get_detail changes.